docs/index.rst

```diff
@@ -207,6 +207,47 @@ with 3 items while processing a request for `/social/list`.
     ...     assert current_menu.submenu('account.list').active
     ...     current_menu.children
 
+
+Flask-Classy
+============
+
+Flask-Classy is a library commonly used in Flask development and gives 
+additional structure to apps which already make use of blueprints. 
+
+Using Flask-Menu with Flask-Classy is rather simple:
+
+.. code-block:: python
+
+    from flask_classy import FlaskView
+    import flask_menu as menu
+
+    class MyEndpoint(FlaskView):
+        route_base = '/'
+
+        @menu.classy_menu_item('frontend.account', 'Home', order=0)
+        def index(self):
+            # Do something.
+            pass
+
+
+As you can see, instead of using the `@menu.register_menu` decorator, we use 
+classy_menu_item. All usage is otherwise the same to `register_menu`, however 
+you do not need to provide reference to the blueprint/app.
+
+You do have to register the entire class with flask-menu at runtime however.
+
+.. code-block:: python
+
+
+    from flask.ext import menu
```

Comment:
@jirikuncar Whilst i'm at it, would you like this done for the entire document, rather than just my additions? (Not sure if you mean globally by my changes, or the entire document)